db/migrations/migrations_test.go

```diff
@@ -502,41 +504,28 @@ type migrator struct {
 }
 
 func (m migrator) runMigration(t *testing.T, idx string) {
```

Comment:
Can we change the function signature of `runMigration` to take in the idx as a `uint`?